bedrock/newsletter/templates/newsletter/developer.html

```diff
@@ -28,14 +28,14 @@
       <div id="tabzilla">
```

Comment:
This may not have been part of the original change request - but this is a very old page, and the `tabzilla` tab here is our old brand logo. I'd suggest either removing it, or updating to the new Mozilla logo while you're here?

Reply:
I'm going to go ahead and remove it for now. Having a hard time finding an example of the new logo that is not in a nav context.